### PR TITLE
Add back the stefina test

### PR DESCRIPTION
**Pull request by ThrawnCA**

### CODE REVIEW

**Comment by duttonw** [comment]
why is this failing??

**Comment by ThrawnCA** [comment]
> why is this failing??

I don't know. The new test seems to have syntax almost identical to all the other tests, yet it's failing the lint check while they pass.

**Comment by duttonw** [comment]
try pulling the branch and doing a rebase ontop of develop. it could be that the lint test config got stricter. 

I/we have been caught up on this with food pantry lint testing of javascript files where the lint config was updated and the merge somehow did not show that major difference between the two branches.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #based on https://raw.githubusercontent.com/ckan/ckanext-scheming/master/.github/workflows/test.yml
 # alternative https://github.com/ckan/ckan/blob/master/contrib/cookiecutter/ckan_extension/%7B%7Bcookiecutter.project%7D%7D/.github/workflows/test.yml
 name: Tests
-on: [push, pull_request]
+on: [push]
```
Comment (duttonw) [inline .github/workflows/test.yml:5]: why not run on pull_requests?

File: ckanext/xloader/plugin.py [hunk 2]
```diff
@@ -127,22 +130,22 @@ def notify(self, entity, operation=None):
...
 
-                try:
-                    task = p.toolkit.get_action('task_status_show')(
-                        context, {
-                            'entity_id': entity.id,
-                            'task_type': 'xloader',
-                            'key': 'xloader'}
-                    )
+                # try:
+                #     task = p.toolkit.get_action('task_status_show')(
```
Comment (duttonw) [inline ckanext/xloader/plugin.py:134]: is this not required?

**Review by duttonw — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (ThrawnCA) [reply .github/workflows/test.yml:5]: There's no real need when it's tested on every push. The results will still appear on the pull request, it just won't launch a redundant run.

**Pull request merged**